keys.go

```diff
@@ -26,11 +26,25 @@ type KeyCapabilities struct {
 	} `json:"devices"`
 }
 
-// CreateKeyRequest describes the definition of an authentication key to create.
+// CreateKeyRequest describes the definition of an authentication key or oauth client to create.
+// Deprecated: Use CreateAuthKeyRequest instead.
 type CreateKeyRequest struct {
+	Description   string          `json:"description"`
 	Capabilities  KeyCapabilities `json:"capabilities"`
 	ExpirySeconds int64           `json:"expirySeconds"`
+}
+
+type CreateAuthKeyRequest struct {
```

Comment:
Nit: I don't know that it's worth defining a separate type, just having a new method seems good enough to me.